scenario-specs/22-dao-narrative-building.md

```diff
@@ -0,0 +1,46 @@
+# DAO Narrative Building  
+## Description  
+Make it easier for individuals to stay in tune with large collective's activities and thought patterns.  
+
+## Motivation  
+As groups of diverse individuals, all spread across the world, start to collaborate and govern a shared system, keeping these individuals all in tune with each other becomes more and more complex and equally important. Having people on the same page helps with cohesion in thought and execution on initiatives. You can think of this as a collective's net efficiency rating.  
+
+Traditionally, this collaboration takes place in the form of:  
+* Short Form Text (Chat Rooms, Slack, Telegram, etc)  
+* Long Form Text (Forums, Issue Tracking, Documentation, Blog Posts, etc)  
+* Voice / Video Calls  
+* Other Platform Activity (DAO Activity, Repository Activity, Social Media Activity, etc)  
+
+If someone wanted to stay in-tune with the collective's activity and thought patterns, they'd find themselves scouring these different platforms. It's important to remember that this is the ambitious collaborator. The other alternative is to just ignore things, leading to further isolation from the group, which is a loss for the individual AND the group.  
+
+People should not be required to read through a long and potentially noisy chat log to see what's going on in a DAO. If distributed governance protocols can scale (ex: DAOstack's Holographic Consensus), the the tools for communicating and consuming communications need to scale as well.  
+
+## Example Scenarios  
+### Chat To Archive  
+**Goals: Declutter Chat, Grow Knowledge Archive**  
+**Description:** 5 group members are discussing the specifics of a new feature in chat, and they want to retain what was discussed. They select the ~50 messages and export them to an archive. Here they can further collaborate on documenting this vision, maybe through a collaborative document editor or a forum. The original messages could be retained as the "source" material within the archive.  
+
+Additionally you could:  
+* Incentivize contributions to the archive, scaling reward based on community response.  
+* Have the chat history clear after N days, further enforcing this workflow of retaining valuable communication.  
+* Replace the original messages in chat w/ a link to the new archive item, further decluttering chat.  
+
+### Call To Archive  
+**Goals: Retain Call Activity, Grow Knowledge Archive**  
+**Description:** Every week the DAO has a call where they discuss the past week. Recordings are made and uploaded to the archive. Attendees are recorded. Notes are taken and appended to the recording. Tags within the video timeline are added for easy reference.  
+
```

Comment:
I really like this idea, are we committed to the chats, etc being public always during the "work" period?